src/praisonai/praisonai/bots/_presentation_renderer.py

```
@@ -0,0 +1,379 @@
+                    for option in block.options[:100]:  # Limit options
+                        label = option.label
+                        if option.emoji:
+                            label = f"{option.emoji} {label}"
+                        label = label[:64]
+                        
+                        inline_keyboard.append([{
+                            "text": label,
+                            "callback_data": f"select:{block.action_id}:{option.value[:32]}"
+                        }])
+        
+        result = {"text": "\n\n".join(text_parts) if text_parts else "​"}  # Zero-width space if no text
```

Ruff (0.15.17)

[error] Invalid unescaped character zero-width-space, use "\u200B" instead

Replace with escape sequence

(PLE2515)